libp2p/Service.cpp

```diff
@@ -150,6 +150,17 @@ void Service::heartBeat()
     });
 }
 
+/// reset whitelist: stop session which is not in whitelist
+void Service::resetWhitelist()
+{
+    std::cout << "reset white list start" << std::endl;
+    RecursiveGuard l(x_sessions);
+    std::this_thread::sleep_for(std::chrono::milliseconds(5000));
```

Comment:
what do you want to do here, why sleep 5 seconds?

Reply:
I have the same doubts

Reply:
Each PR can only contain 250 lines. I just PR trigger logic here, implementation will be requested at next PR.